meetings/2020-12-15.rst

```diff
@@ -0,0 +1,76 @@
+    This is perhaps a desirable design as we go forward.
+      
+  * One of the things we are trying to do with our standardization effort is to build a foundation to express machine-representable relationships as a platform for translating between different application-specific ontologies (e.g., used for different databases)
+
+  * To some degree, we may want to think about "data formats" as separate from the underlying ontologies.
+    
+    - Ontologies are more about semantics than representation.
+    
+    - Ontologies are sometimes explicitly connected to specific data formats and sometimes not.
+    
+    - Data formats typically have an ontology tied to them, such as CIF.
+
+  * A related observation is that in data formats such as CIF, it is often a goal to unify data on a common format with restrictions to what variation of data representation can be used,
+    e.g., a few or a single way to represent a crystal structure. 
+    Depending on the goal with an ontology it may not be designed with the same desire, 
+    but rather one may aim to define many related concepts and alternatives for representation.
+    
+    - We may want to consider where our effort is positioned in this. To standardize for integration between databases, we may want to be restrictive, 
+      and accommodate the very wide range of alternative representations primarily via the definition of transformations to/from the ontology from other external ontologies (e.g., database-specific ontologies).
+
+  * CIF has a concept of "composite material" that is different from ours; a composite of overlapping repeating lattices.
+    We may want to consider adding this model and carefully consider naming in relation to our "Composite materials model."
+
+- Discussion about the division in "homogenous" and "inhomogenous" materials models
+
+  * Do we want to build this type of classifications into the "main tree" hierarchy, or not? 
+  
+    - Is there a point to adding extra steps in the "hierarchy of models"? Or, is there a desirable simplicity in keeping all specific materials models as the direct sublevel below "Materials model"? 
+      
+    - We can still have 'separate' classification concepts (e.g., entities) that we tie to the models, e.g., homogenous and inhomogenous, that would allow ontology-based reasoning on these concepts.
+
+- A similar question about hierarchical representation is about the "single structure" materials model we presently have in the prototype ontology.
+  This naming suggests a concept that encompasses many "submodels" that all fall under 'one structure.'
+  
+  * This may mostly be a naming thing in the prototype ontology. Our description relates more closely to "Single crystal," so perhaps we just need to rename "Single structure materials model" -> "Single crystal materials model."
+  
+  * We may want to branch out a few other models: Powder models? (Single material powder? Multiple materials powder?) Monolithed structures.
```

Comment:
Since this is meeting notes and not a place to discuss the points further, I'll not add a note here, but I have some ideas on how powders could be added. Well, one idea.

Reply:
Maybe post your thoughts as an issue?

Technically I'm fine with the discussion part of the PR being used to discuss things that come up in relation to the meeting - and last time I added those notes to the end of the meeting notes before committing it. But, new models will likely end up a new PR, so it makes sense to take it as an issue, I think.